### PR TITLE
lib/standard/bytes: Added pop method

### PR DESCRIPTION
**Pull request by R4PaSs**
The method was missing from Bytes, I needed it so I implemented it, and here's the associated PR.

### CODE REVIEW

**Comment by privat** [comment]
+1

**Comment by privat** [comment]
you should document properties in lib/standard/bytes.nit

**Comment by R4PaSs** [comment]
I guess I should yep

**Comment by R4PaSs** [comment]
K done reroll fixed missing doc on bytes

File: lib/standard/bytes.nit [hunk 1]
```diff
@@ -96,6 +97,16 @@ class Bytes
 		end
 	end
 
+	#     var b = new Bytes.empty
+	#     b.append([0x41u8, 0x41u8, 0x18u8])
+	#     b.pop
+	#     assert b.to_s == "AA"
+	redef fun pop do
+		assert length >= 1
```
Comment (Morriar) [inline lib/standard/bytes.nit]: assert not is_empty?

**Comment by Morriar** [comment]
+1

File: lib/standard/bytes.nit (hunk 1 above)
Comment (R4PaSs) [inline lib/standard/bytes.nit]: That would work too yes, should I change it or do I have permission to leave it as is ?

Comment (privat) [inline lib/standard/bytes.nit]: assert not_empty?

Comment (R4PaSs) [inline lib/standard/bytes.nit]: Cf reply from above

Comment (privat) [inline lib/standard/bytes.nit]: @R4PaSs follow your conscience

Comment (R4PaSs) [inline lib/standard/bytes.nit]: The one with the horns or the one with an aureole ?

**Pull request merged**